Documentation/rust/quick-start.rst

```diff
@@ -0,0 +1,47 @@
+.. _rust_quick_start:
+
+Quick Start
+===========
+
+This document describes how to get started with Rust kernel development.
+
+Requirements
+------------
+
+A nightly Rust toolchain (at least ``rustc``, ``cargo`` and ``rustfmt``) is required. In the future, this restriction will be lifted. If you are using ``rustup``, run::
```

Comment:
We should probably pick some nightly MSRV. The one in the linux-kernel-module-rust `.travis.yml` is as good a place to start as any :-)

Reply:
Ah, you meant for those with old nightlies! Agreed!

Reply:
New wording.